go/api/bitmap_commands_test.go

```diff
@@ -11,9 +12,9 @@ import (
 func ExampleGlideClient_SetBit() {
 	var client *GlideClient = getExampleGlideClient() // example helper function
 
-	client.SetBit("my_key", 1, 1) // initialize bit 1 with a value of 1
+	client.SetBit(context.TODO(), "my_key", 1, 1) // initialize bit 1 with a value of 1
```

Comment:
We should use `context.Background()` instead of `context.TODO()`. While they both work exactly the same, the `TODO` implementation is meant to be a placeholder when you don't know which type of context you might need. You can read about this more at https://dev.to/zakariachahboun/are-you-confused-about-contexttodo-vs-contextbackground-heres-what-you-need-to-know-2h46

Reply:
Updated all in 9aaabc81b
![image](https://github.com/user-attachments/assets/fe6b18ea-7bd0-4469-af07-0b92a95d2b64)